Header header

float32 x
float32 y
float32 z

float32 yaw
float32 pitch
float32 roll

geometry_msgs/Pose[] tag_location 
#T0 ~ T7